tests/test_github_all_fields.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from tap_tester import runner, connections, menagerie
+
+from base import TestGithubBase
+
+
+class TestGithubAllFields(TestGithubBase):
+    """Test that with all fields selected for a stream automatic and available fields are  replicated"""
+
+    @staticmethod
+    def name():
+        return "tap_tester_github_all_fields"
+
+    def test_run(self):
+        """
+        Ensure running the tap with all streams and fields selected results in the
+        replication of all fields.
+        - Verify no unexpected streams were replicated
+        - Verify that more than just the automatic fields are replicated for each stream.
+        """
+        # BUG TDL-16672
+        # The excluded streams are not honoring all fields selection
```

Comment:
I commented on this bug, I am not sure this is accurate.

Reply:
removed as bug was identified in tap